odoo/addons/test_main_flows/static/tests/tours/main_flow.js

```diff
@@ -865,100 +865,96 @@ tour.stepUtils.mobileModifier(tour.stepUtils.autoExpandMoreButtons('.o_control_p
     content: _t('See Project'),
     position: 'right',
 },
-tour.stepUtils.autoExpandMoreButtons('.o_control_panel .breadcrumb:contains("the_flow.project")'),
+tour.stepUtils.toggleHomeMenu(),
+...tour.stepUtils.goToAppSteps("project.menu_main_pm", 'Go to the Project app.'),
 {
-    trigger: '.oe_button_box .oe_stat_button:has(span:contains("Recorded"))',
-    extra_trigger: '.o_form_readonly',
-    content: _t('See Timesheets'),
+    trigger: ".o_kanban_record:has(span:contains('the_flow.project'))",
+    content: _t('Open project'),
     position: 'bottom',
 }, {
-    mobile: true,
-    trigger: '.o_enable_searchview',
-    content: _t('Open the search field'),
+    trigger: ".o_kanban_record:has(span:contains('the_flow.service')) .o_dropdown_kanban",
```

Comment:
Why so many changes to this tour? Can you not access the same view as before through the kanban burger menu?